coalib/bears/requirements/PythonRequirement.py

```diff
@@ -83,6 +103,15 @@ def multiple(cls, *args):
                     reqs.append(cls(name, version),)
                 except ValueError:
                     raise TypeError('The tuple must have 2 elements.')
+            elif isinstance(requirement, list):
+                if len(requirement) == 1:
+                    reqs.append(cls(requirement),)
+                else:
+                    try:
+                        name, version = requirement
+                        reqs.append(cls(name, version),)
+                    except ValueError:
+                        raise TypeError('The list must have max 2 elements.')
```

Comment:
basically when writing python you usually don't want to know what an object is, you would rather do stuff with it and if it can't do it you expect it to raise an exception. This way people could write new objects with the same capabilities that are then supported generically since they have the capabilities.